openapi/components/examples/requests/post-orders/server-to-server.json

```
@@ -0,0 +1,24 @@
+{
+    "type":"direct",
+    "gateway":"CREDITCARD",
+    "order_id":"my-order-id-1",
+    "currency":"EUR",
+    "amount":1000,
+    "description":"Test order description",
+    "payment_options":{
+      "notification_url":"https://www.example.com/client/notification?type=notification",
+      "redirect_url":"https://www.example.com/client/notification?type=redirect",
+      "cancel_url":"https://www.example.com/client/notification?type=cancel",
+      "close_window":true
+    },
+    "customer":{
+      "locale":"nl_NL",
+      "ip_address":"123.123.123.123"
+    },
+    "gateway_info":{
+      "card_number":"4111111111111111",
+      "card_holder_name":"Holder Name",
+      "card_expiry_date":"1612",
```

```suggestion
      "card_expiry_date":"2612",
```
This card will be declined because the expiry date has passed.